typing-app/src/components/templates/GameResult.tsx

```diff
@@ -86,9 +86,14 @@ const GameResult: React.FC<GameResultProps> = ({ nextPage, resultScore }) => {
             {resultScore.accuracy.toFixed(1)}%
           </Text>
         </GridItem>
-        <GridItem colSpan={4} rowSpan={2} colStart={4} rowStart={9} className={styles.centerText}>
-          <Button onClick={nextPage} colorScheme="blue" size="lg" w="100%" h="90%">
-            次へ
+        <GridItem colSpan={4} rowSpan={2} colStart={2} rowStart={9} className={styles.centerText}>
+          <Button onClick={() => window.location.href = '/'} colorScheme="red" size="lg" w="80%" h="90%">
```

Comment:
Next.jsのベストプラクティスに従いたいので、遷移方法について指摘させてください。

まず
```typescript
import { useRouter } from "next/navigation";
```
でuseRouterフックをインポートします。

その後
routerオブジェクトを取得し、onClickのイベントハンドラを定義
```typescript
const router = useRouter();

const pushToRoot = () => {
  router.push("/");
}
```


```suggestion
<Button onClick={pushToRoot} colorScheme="red" size="lg" w="80%" h="90%">
  ゲームを終了する
</Button>
```